mesheryctl/internal/cli/root/design/apply.go

```
@@ -118,8 +118,7 @@ mesheryctl design apply [design-name]
 			if !validURL {
 				content, err := os.ReadFile(file)
 				if err != nil {
-					utils.Log.Error(utils.ErrFileRead(errors.Errorf("file path %s is invalid. Enter a valid path ", file)))
-					return nil
+					return errors.Wrap(err, fmt.Sprintf("file path %s is invalid. Enter a valid path", file))
```

We arre using cutsom errors in mesheryctl please return the error logged previously  as is

---

yeah i didnt see that before.working on it.